.travis.yml

```diff
@@ -55,6 +55,15 @@ matrix:
       env: TARGET="Linux-X86_64"   LLDB_TESTS="all"  CLANG="1" COVERAGE="1"
     - os: linux
       env: TARGET="Linux-X86_64"   LLDB_TESTS="all"  CLANG="1" RELEASE="1"
+    # Linux-X86_64 (with GDB tests)
```

Comment:
Partially because I figured that they were likely to break on different changes, and this makes it easier to find the signal. And also partially because it makes the travis env line less long and complicated.